web/src/main/java/org/cbioportal/web/util/ClinicalDataFilterApplier.java

```diff
@@ -46,13 +46,17 @@ public ClinicalDataFilterApplier(PatientService patientService,
             List<Patient> patients = patientService.getPatientsOfSamples(studyIds, sampleIds);
             List<String> patientIds = patients.stream().map(Patient::getStableId).collect(Collectors.toList());
             List<String> studyIdsOfPatients = patients.stream().map(Patient::getCancerStudyIdentifier).collect(Collectors.toList());
+            attributes.forEach(c-> {
+                c.setAttributeId(c.getAttributeId().toUpperCase());
+            });
             clinicalDataList = clinicalDataService.fetchClinicalData(filterClinicalDataType.equals(ClinicalDataType.PATIENT) ?
                     studyIdsOfPatients : studyIds, filterClinicalDataType.equals(ClinicalDataType.PATIENT) ? patientIds : sampleIds,
                 attributes.stream().map(ClinicalDataFilter::getAttributeId).collect(Collectors.toList()),
                 filterClinicalDataType.name(), Projection.SUMMARY.name());
 
             clinicalDataList.forEach(c -> {
-                if (c.getAttrValue().toUpperCase().equals("NAN") || c.getAttrValue().toUpperCase().equals("N/A")) {
+                c.setAttrValue(c.getAttrValue().toUpperCase());
```

Comment:
why are all attribute values converted to upper case?

Reply:
value should be converted to upper case just for conditional checking.

Reply:
mysql is case in sensitive but not persistent layer. if you try 
```
https://www.cbioportal.org/study/summary?filterAttributeId=SEX&filterValues=Male&id=msk_impact_2017
```
and 
```
 https://www.cbioportal.org/study/summary?filterAttributeId=SEX&filterValues=MALE&id=msk_impact_2017
```
you will get a complete different result.

Reply:
you might need to update values(to uppercase) in attributes for it to work

Reply:
yes. I am correcting those lines.